// Copyright 2023 Google. This software is provided as-is, without warranty or representation for any use or purpose.
// Your use of it is subject to your agreement with Google.
 
syntax = "proto2";
 
message SampleData {
  optional int64 Customer_ID = 1;
  optional int64 Customer_Enrollment_Date = 2; //This is a date, but use int64 with Epoch timestamp in microseconds
  optional string Customer_Name = 3;
  optional string Customer_Address = 4;
  optional string Customer_Tier = 5;
  optional string Active_Subscriptions = 6;
  optional string _CHANGE_TYPE = 7;
  optional string _CHANGE_SEQUENCE_NUMBER = 8;
}
